libecdaa/schnorr/schnorr_ZZZ.c

```diff
@@ -81,6 +81,10 @@ int schnorr_sign_ZZZ(BIG_XXX *c_out,
     //      (modular-reduce c', too).
     BIG_XXX c_prime;
     if (basename_len != 0) {
+        // If any of these is non-zero, ALL must be non-zero.
+        if (NULL == basename || 0 == basename_len || NULL == K_out)
```

Comment:
The `0 == basename_len` clause will always be false in this `if` block, but I guess keeping it makes the intent a bit more clear.

Reply:
No, it looks dumb. It was a bad copy-paste, I think.

I just removed it (the `0 == basename_len`)